health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/GeoPode.js

```
@@ -134,64 +182,65 @@ const GeoPode = () => {
                             onClick={() => {
                                 // setShowPopUp(true);
                             }}
-                        /> 
+                        />
                     </div>
-                    <div style={{height:"2rem"}}></div>
+                    <div style={{ height: "2rem" }}></div>
                     <div>
                         {boundaryData.map((item, index) => (
                             <div>
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)